docs/advanced/multi_outputs/prometheus_output.md

```diff
@@ -11,6 +11,7 @@ outputs:
     expiration: 60s # maximum lifetime of metrics in the local cache
     metric-prefix: "" # a string to be used as the metric namespace
     append-subscription-name: false # a boolean, if true the subscription name will be appended to the metric name after the prefix
+    export-timestamps: false # a boolean, enables exporting timestamps as part of the metrics
```

Comment:
since this is the only mentioning on how this field works, maybe we should add more details to it?

`# a boolean, enables exporting timestamps received from the gNMI target as part of the metrics`

the key piece is that the timestamps come from the gNMI target and not computed by Prom on scrape time